content/sensu-go/5.20/reference/backend.md

```diff
@@ -338,6 +340,23 @@ discovery instead of the static `--initial-cluster method`
 
 ### General configuration flags
 
+| annotations|      |
+-------------|------
+description  | Non-identifying metadata to include with event data that you can access with [event filters][9] and [tokens][27]. You can use annotations to add data that is meaningful to people or external tools that interact with Sensu.<br><br>In contrast to labels, you cannot use annotations in [API response filtering][25], [sensuctl response filtering][26], or [dashboard view filtering][54].
```

Comment:
Reading this information back I'm realizing that most of this is only applicable to the agent entity doc. Backend entity labels and annotations will only be used in filters and token substitution (not yet implemented) for backend assets (ex. handler assets and mutator assets). API response filtering, sensuctl response filtering, and dashboard view filtering will not be possible for either field because we do not currently store the backend as an entity.